tests/integration/renderer/test_multi_output.py

```diff
@@ -133,9 +133,7 @@ def set_bc(a):
         dev_tools_hot_reload=False,
     )
 
-    # the UI still renders the output triggered by callback.
-    # The new system does NOT loop infinitely like it used to, each callback
-    # is invoked no more than once.
```

Comment:
I gather we're back to the behavior in this deleted comment now, not the new comment "The new system does NOT trigger callbacks in circular dependencies?"

Reply:
yes, need to 🔪  the 🔪

Reply:
https://github.com/plotly/dash/pull/1254/commits/ca02d5578f13eb39acb111a33d1f4f1f84477282